src/test/java/ooo/oshi/SystemInfoTest.java

```diff
@@ -12,7 +12,7 @@
 import ooo.oshi.software.os.OperatingSystem;
 import ooo.oshi.util.FormatUtil;
 
-public class Test {
+public class SystemInfoTest {
```

Comment:
This `Test` class was never intended to be part of the final result... I've been using it to quickly test code changes before getting around to setting up proper CI testing.  Might even want to document it as such (e.g., "feel free to modify this class for ad hoc testing and debugging").

I don't mind switching it to the test directory (please update the pom to remove the commented test scope for slf4j.simple) but would prefer the "SystemInfoTest" do a much more complete test of everything that we add to, and use logging rather than System.out like the OSHI version.